admin/tests/helpers/test-init.js

```diff
@@ -9,10 +9,15 @@ export async function createAuthenticateSession({ userId }) {
   });
 }
 
-export function authenticateAdminMemberWithRole({ role }) {
+export function authenticateAdminMemberWithRole({
+  isSuperAdmin = true,
+  isCertif = false,
+  isSupport = false,
+  isMetier = false,
+}) {
```

Comment:
On pourrait setter par defaut a un objet vide et donc tout les autres changement tu pourrais plutot le modifier a `await authenticateAdminMemberWithRole()(server);`
```suggestion
export function authenticateAdminMemberWithRole({
  isSuperAdmin = true,
  isCertif = false,
  isSupport = false,
  isMetier = false,
} = {}) {
```

Reply:
vu ensemble : on enlève les valeurs par défaut comme ça on aura jamais besoin de mettre isSuperAdmin à false quand on veut un autre role